includes/options/class-amp-options-menu.php

```diff
@@ -150,15 +150,15 @@ public function render_theme_support() {
 						<p><?php esc_html_e( 'Your active theme is known to work well in paired or native mode.', 'amp' ); ?></p>
 					</div>
 				<?php endif; ?>
-				<dl>
+				<dl>					
```

Comment:
Looks like there is trailing whitespace introduced on some lines which should get trimmed.

Reply:
Sorry, this was my first contribution, I fixed it now.

Reply:
Welcome and thank you for your contribution, @AdelDima.